gnovm/tests/files/recover10.gno

```diff
@@ -0,0 +1,7 @@
+package main
+
+func main() {
+	defer recover()
+
+	panic("ahhhhh")
+}
```

Comment:
When running this [code in the go playground](https://go.dev/play/p/5CVzJmkO3lm), the panic is not caught, so you have to add this assertion. 

```suggestion
}

// Error:
// panic: ahhhhh
```

Unfortunately, the test fails after that, the recover seems to catch the panic when it shouldn't. This is probably a different case to handle as it is also broken on the `master` branch. You can probably fix this later, but then you should move it to the challenges directory.

Reply:
fixed